src/styles/blocks/default-styles.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
not a good practice to reset indent for every element in page, just do it for those you really need